protocol/scripts/deploys/mainnet/v2/contract-addresses.ts

```diff
@@ -0,0 +1,429 @@
+            EXECUTOR_MSIG: string,
+            RESCUER_MSIG: string,
+            UNDERLYING_GNOSIS_SAFE: string,
+            CIRCUIT_BREAKERS: {
+                DAI: string,
+                TEMPLE: string,
+            },
+        },
+        RAMOS_STRATEGY: {
+            ADDRESS: string,
+            EXECUTOR_MSIG: string,
+            RESCUER_MSIG: string,
+            CIRCUIT_BREAKERS: {
+                DAI: string,
+                TEMPLE: string,
+            },
+        },
+        TLC_STRATEGY: {
+            ADDRESS: string,
+            EXECUTOR_MSIG: string,
+            RESCUER_MSIG: string
+        },
+    },
+    EXTERNAL: {
+        MAKER_DAO: {
+            DAI_TOKEN: string,
+            DAI_JOIN: string,
+            POT: string,
+        },
+        BALANCER: {
+            VAULT: string,
+            HELPERS: string,
+            TOKEN: string,
+        },
+    }
+}
+
+const V2_DEPLOYED_CONTRACTS: {[key: string]: ContractAddresses} = {
+    localhost: {
+        CORE: {
+            TEMPLE_TOKEN: '0x470ebf5f030ed85fc1ed4c2d36b9dd02e77cf1b7',
+            CIRCUIT_BREAKER_PROXY: '0xE2b5bDE7e80f89975f7229d78aD9259b2723d11F',
+            // GNOSIS_SAFE_GUARD: '',
+            EXECUTOR_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+            RESCUER_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+        },
+        TREASURY_RESERVES_VAULT: {
+            ADDRESS: '0x4ea0Be853219be8C9cE27200Bdeee36881612FF2',
+            D_USD_TOKEN: '0x46d4674578a2daBbD0CEAB0500c6c7867999db34',
+            D_TEMPLE_TOKEN: '0x9155497EAE31D432C0b13dBCc0615a37f55a2c87',
+            TPI_ORACLE: '0xC6c5Ab5039373b0CBa7d0116d9ba7fb9831C3f42',
+        },
+        TEMPLE_LINE_OF_CREDIT: {
+            ADDRESS: '0x05bB67cB592C1753425192bF8f34b95ca8649f09',
+            CIRCUIT_BREAKERS: {
+              DAI: '0xa85EffB2658CFd81e0B1AaD4f2364CdBCd89F3a1',
+              TEMPLE: '0x8aAC5570d54306Bb395bf2385ad327b7b706016b',
+            },
+            INTEREST_RATE_MODELS: {
+              LINEAR_WITH_KINK: '0x38628490c3043E5D0bbB26d5a0a62fC77342e9d5',
+            },
+        },
+        RAMOS: {
+            AUTOMATION_EOA: '0x67ae2161449cf3c5528fea969beba3e4f3288c61',
+            TEMPLE_DAI: {
+                ADDRESS: '0x88D1aF96098a928eE278f162c1a84f339652f95b',
+                POOL_HELPER: '0x7Ce73F8f636C6bD3357A0A8a59e0ab6462C955B0',
+                AURA_STAKING: '0x6345e50859b0Ce82D8A495ba9894C6C81de385F3',
+                AURA_TOKEN: '0xC0c293ce456fF0ED870ADd98a0828Dd4d2903DBF',
+                AURA_BOOSTER: '0xA57b8d98dAE62B26Ec3bcC4a365338157060B234',
+                FEE_COLLECTOR: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                BPT_TOKEN: '0x173063a30e095313eee39411f07e95a8a806014e',
+                POOL: '0x8bd4a1e74a27182d23b98c10fd21d4fbb0ed4ba00002000000000000000004ed',
+                POOL_ID: '79',
+                REWARDS: '0x987c70086239e1a1e9f242ca19fb5a270d4e72b2'
```

Comment:
various new addresses here, some were deployed locally, some I took them from mainnet, please let me know if this is correct, I have a feeling I may have missed something here

Reply:
See below under `mainnet` - the addresses would be the same for those ones in localhost